packages/gensx/package.json

```diff
@@ -44,7 +44,7 @@
     "@types/chokidar": "^2.1.7",
     "@types/ini": "^4.1.1",
     "@types/node": "catalog:packages",
-    "@types/react": "^18.2.0",
+    "@types/react": "catalog:",
```

Comment:
we should probably just use whatever makes sense for the project instead of sharing this across all projects using the catalog. I assume we will update react separately per project.

Reply:
sounds good, i will just update them individually